<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Odonto</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastros">
    <property name="title">
     <string>Cadastros</string>
    </property>
    <addaction name="actionIndica_es"/>
    <addaction name="actionAlunos"/>
    <addaction name="actionPacientes"/>
    <addaction name="actionPanoramicas"/>
    <addaction name="actionTomografias"/>
    <addaction name="actionCefalogramas"/>
   </widget>
   <addaction name="menuCadastros"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIndica_es">
   <property name="text">
    <string>Indicações</string>
   </property>
  </action>
  <action name="actionAlunos">
   <property name="text">
    <string>Alunos</string>
   </property>
  </action>
  <action name="actionPacientes">
   <property name="text">
    <string>Pacientes</string>
   </property>
  </action>
  <action name="actionPanoramicas">
   <property name="text">
    <string>Panorâmicas</string>
   </property>
  </action>
  <action name="actionTomografias">
   <property name="text">
    <string>Tomografias</string>
   </property>
  </action>
  <action name="actionCefalogramas">
   <property name="text">
    <string>Cefalogramas</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
